### PR TITLE
Hunting queries windows event batch5

### PR DESCRIPTION
**Pull request by yaelrbergman**
Required items, please complete

Change(s):
Add Windows Event Data to the following Hunting Queries:

Hunting Queries/SecurityEvent/HostsWithNewLogons.yaml
Hunting Queries/SecurityEvent/UserAccountAddedToPrivlegeGroup.yaml
Hunting Queries/SecurityEvent/UserAccountCreatedDeleted.yaml
Hunting Queries/SecurityEvent/VIPAccountFailedLogons.yaml
Hunting Queries/SecurityEvent/WindowsSystemTimeChange.yaml
Hunting Queries/ThreatIntelligenceIndicator/FileEntity_SecurityEvent.yaml

Reason for Change(s):
Part of adding Windows Event data source project

Version Updated:
No
Testing Completed:
Yes
Checked that the validations are passing and have addressed any issues that are present:
Yes

### CODE REVIEW

File: Hunting Queries/SecurityEvent/HostsWithNewLogons.yaml [hunk 1]
```diff
@@ -19,14 +22,28 @@ query: |
   let endtime = todatetime('{{EndTimeISO}}');
```
Comment (oshezaf) [inline Hunting Queries/SecurityEvent/HostsWithNewLogons.yaml:22]: FYI only: I don't think this is actually going to work.... unless someone edits it manually :-( Not sure how to handle.

File: Hunting Queries/SecurityEvent/VIPAccountFailedLogons.yaml [hunk 2]
```diff
@@ -14,12 +17,22 @@ query: |
...
   let List = datatable(VIPUser:string, Domain:string)["Bob", "Domain", "joe", "domain", "MATT", "DOMAIN"];
-  List | extend Account = strcat(Domain,"\\",VIPUser) | join kind= inner (
-  SecurityEvent 
+  List | extend Account = strcat(Domain,"\\",VIPUser) | join kind= inner ( union isfuzzy=true 
+  (SecurityEvent 
   | where EventID == "4625"
   | where AccountType == "User"
   | where LogonType == "2" or LogonType == "3"
-  ) on Account 
+  | summarize StartTimeUtc = min(TimeGenerated), EndTimeUtc = max(TimeGenerated), FailedVIPLogons = count() by LogonType, Account
```
Comment (oshezaf) [inline Hunting Queries/SecurityEvent/VIPAccountFailedLogons.yaml:25]: Lines 25-27 accidentally left in while actually replaced by 36-38.

File: Hunting Queries/ThreatIntelligenceIndicator/FileEntity_SecurityEvent.yaml [hunk 3]
```diff
@@ -25,12 +28,21 @@ query: |
   | where Active == true
```
Comment (oshezaf) [inline Hunting Queries/ThreatIntelligenceIndicator/FileEntity_SecurityEvent.yaml:28]: This is an incorrect way to use the TI table. If you converted any in the past, you know it needs to have a summarize clause between lines 27 and 28. In this case I suggest you do add it.

File: Hunting Queries/ThreatIntelligenceIndicator/FileEntity_SecurityEvent.yaml [hunk 4]
```diff
@@ -25,12 +28,21 @@ query: |
...
   | where isnotempty(FileName)
   // using innerunique to keep perf fast and result set low, we only need one match to indicate potential malicious activity that needs to be investigated
-  | join kind=innerunique (
-    SecurityEvent 
+  | join kind=innerunique ( union isfuzzy=false 
+    (SecurityEvent 
     | where TimeGenerated between(starttime..endtime)
     | where EventID in ("4688","8002","4648","4673")
```
Comment (oshezaf) [inline Hunting Queries/ThreatIntelligenceIndicator/FileEntity_SecurityEvent.yaml:34]: This raises an issue with a change I asked you do in the past, and might have been a mistake: namely adding 4688 to detection rules that where about process. I think we should revert those.

**Review by oshezaf — CHANGES_REQUESTED** [review]

File: Hunting Queries/ThreatIntelligenceIndicator/FileEntity_SecurityEvent.yaml (hunk 4 above)
Comment (yaelrbergman) [reply Hunting Queries/ThreatIntelligenceIndicator/FileEntity_SecurityEvent.yaml:34]: @oshezaf I added 4688 event filtering only to detections that conditioned on CommandLine not by process

Comment (oshezaf) [reply Hunting Queries/ThreatIntelligenceIndicator/FileEntity_SecurityEvent.yaml:34]: Great. So we were smarter than I remembered :-)

**Comment by vmanojreddy** [comment]
@aprakash13  Please review hunting queries. thanks.

**Comment by vmanojreddy** [comment]
@oshezaf Please check and approve the changes, thanks.

**Comment by vmanojreddy** [comment]
@oshezaf can you please take a look and provide your feedback. thanks!!

**Comment by v-mchatla** [comment]
Hi @oshezaf
Can we close this and reopen whenever required as you have labelled it as differed.
Thanks

**Comment by v-mchatla** [comment]
Hi @oshezaf
Thanks for closing the differed PRs. 
Can you also close remaining differed PRs (#4157, #3758, #2699, #4253, #4332).
Thanks

**Comment by oshezaf** [comment]
I closed 2 and left 2 intentionally open. Note that #4332<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2FAzure-Sentinel%2Fpull%2F4332&data=05%7C01%7C%7C1caa38654a0d40eb74f608da918d8f25%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637982334417732182%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=Kr%2FD5k6bxdMrFzGOwySTA%2FKH30m3c8p%2BTupbEuzRL2A%3D&reserved=0> is not mine.

**Pull request closed without merging**